### DISCUSSION TITLE
Algolia search set up

### DISCUSSION THREAD

**Discussion by rastislavcore** [vuepress-theme-hope/vuepress-theme-hope#1918] [Q&A] [answered]
Can you point me to the manual, where is a description, of how to set up the Algolia search?

Thank you in advance.

**Comment by Mister-Hope** [comment, marked as answer]
I think I had added enough in https://vuepress-theme-hope.github.io/v2/guide/feature/search.html.

I personally do not think I should provide help with algolia, if you need any, you'd better ahead to docsearch discord channel or other places.
